pkg/apis/azure/types_virtualmachine.go

```diff
@@ -35,10 +35,8 @@ type VirtualMachineSpec struct {
 	Hostname string
 	// ProviderID is the provider ID of the Kubernetes node for this virtual machine.
 	ProviderID string
-	// Ready is whether the Kubernetes node for this virtual machine is ready.
-	Ready bool
-	// Unreachable is whether the Kubernetes node for this virtual machine is unreachable.
-	Unreachable bool
+	// NotReadyOrUnreachable is whether the Kubernetes node for this virtual machine is either not ready or unreachable.
+	NotReadyOrUnreachable bool
```

Comment:
I would rename this to UnreachableOrNotReady. Otherwise it's unclear if it is Not(Ready)OrUnreachable or Not(ReadyOrUnreachable).